repose-aggregator/commons/configuration/src/test/resources/unmarshallerValidator/limits/limits.xsd

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yeah I hate this super hard. But we're supposed to drop all this in a later release, so it's not as bad.